internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/consumer/TestChangelogConsumer.java

```diff
@@ -102,7 +102,7 @@
 
 
 public class TestChangelogConsumer {
```

Comment:
Shall we add a CC test with a materialized view either here or in `TestMaterializedViewEndToEnd` to do some simple verification? Like hybrid writes can be captured in the expected view partitions.

Reply:
ehhhh.  Unfortunately until we do the source partition thing this isn't really doable.  The MaterializedView enforces us to have a different partition count or different partitioner.  This means the record count is gonna be wonky because we'll filter messages incorrectly.  So I'd rather wait until we square that part away.